components/mpas-ocean/src/tracer_groups/Registry_activeTracers.xml

```diff
@@ -65,8 +65,8 @@
 				<var name="temperature" array_group="activeGRP" units="degrees Celsius"
```

Comment:
```suggestion
				<var name="temperature" array_group="activeGRP" units="C"
```
This may need to be changed elsewhere, too, so it would be worth doing a search.